includes/class-ssp-frontend.php

```diff
@@ -116,6 +119,17 @@ public function __construct( $file, $version ) {
 		add_action( 'wp_enqueue_scripts', array( $this, 'load_scripts' ) );
 	}
 
+	public function html5_player_conditional_scripts(){
+		global $largePlayerInstanceNumber;
+		if( (int) $largePlayerInstanceNumber > 0 ){
```

Comment:
If `$largePlayerInstanceNumber ` is greater than 0, it means we have at least one HTML5 player on the page, so we defer our scripts to the footer area, it is the only solution given the order of execution and when we know when a player is present.